Source/LibationFileManager/InteropFactory.cs

```diff
@@ -61,18 +68,13 @@ private static string getOSConfigApp()
             var here = Path.GetDirectoryName(Environment.ProcessPath);
 
             // find '*ConfigApp.exe' files
-            var exes =
-                Directory.EnumerateFiles(here, $"*{CONFIG_APP_ENDING}", SearchOption.TopDirectoryOnly)
+            var appName =
+                Directory.EnumerateFiles(here, $"*{CONFIG_APP_ENDING}*", SearchOption.TopDirectoryOnly)
```

Comment:
How will the final star help? Is it ever .dll.other ?

Reply:
Oops. That was a mistake. Fixed.